src/utils/formatAmount.ts

```diff
@@ -0,0 +1,7 @@
+export const formatAmount = (amount: number | string) => {
```

Comment:
**suggestion (non-blocking)** maybe a smol comment on what this funciton does would be helpful